requirements/deploy.txt

```diff
@@ -177,17 +177,17 @@ dparse==0.5.2
     # via -r requirements/pip.txt
 drf-extensions==0.7.1
     # via -r requirements/pip.txt
-drf-flex-fields==0.9.8
+drf-flex-fields==1.0.0
```

Comment:
Nothing to worry about: https://github.com/rsinger86/drf-flex-fields#100-august-2022